CHANGELOG.md

```diff
@@ -5,9 +5,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-### Fixed
+## [3.0.0] - 2023-11-16
+
+### Changed
```

Comment:
Why don't you write some dependency updates?
https://github.com/cybozu-go/sabakan/pull/266
https://github.com/cybozu-go/sabakan/pull/273

Reply:
Thank you. I added these PRs.